src/style.css

```diff
@@ -2,2 +1,93 @@
+body,
+p,
+h1 {
   margin: 0;
 }
+
+ul {
+  margin: 0;
+  padding-inline-start: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+  line-height: 15px;
+  width: 100vw;
+
+  --main-lc: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.logo img {
+  display: block;
+}
+
+.nav {
+  margin-right: 50px;
+  display: flex;
+}
+
+.nav_list {
+  list-style: none;
+  display: inline-block;
+}
+
+.nav_item {
+  display: inline-block;
+  text-transform: uppercase;
+  padding-left: 17px;
+  align-self: center;
+}
+
+.nav_item:first-child {
+  padding-left: 0;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+  text-align: center;
+  height: 60px;
+  position: relative;
+  display: block;
+  padding: 22.5px 0;
+}
+
+.nav_link[data-qa='hover']:hover {
+  color: var(--main-lc);
+}
```

Comment:
The '.nav_link[data-qa='hover']:hover' selector changes the color on hover. Ensure the 'data-qa' attribute is used correctly, typically for testing purposes, and verify its necessity.